angular/projects/spark-angular/src/lib/components/sprk-footer/sprk-footer.component.ts

```diff
@@ -25,7 +25,7 @@ import {
             *ngIf="globalLinks"
             class="sprk-o-Stack__item sprk-o-Stack__item--three-tenths@m sprk-o-Stack sprk-o-Stack--misc-b sprk-o-Box sprk-u-prh"
           >
-            <h3 class="sprk-o-Stack__item sprk-b-TypeBodyOne">
+            <h3 class="sprk-o-Stack__item sprk-b-TypeBodyOne sprk-c-Footer__type">
```

Comment:
I wonder if there's a clearer name for `sprk-c-Footer__type`

Looking at the code is seems to dictate the Footer's type color.
Would `sprk-c-Footer__content` or `sprk-c-Footer__text` be more clear?

Reply:
![image](https://user-images.githubusercontent.com/4342363/88946372-2dc68d80-d25d-11ea-915f-b0a6eeaccd1b.png)

Reply:
We were spit balling on name ideas and I quickly just came up with __type, I like `sprk-c-Footer__text` better though!!! @RVMendoza